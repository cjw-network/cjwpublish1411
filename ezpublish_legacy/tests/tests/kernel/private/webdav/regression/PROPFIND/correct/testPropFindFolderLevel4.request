<?php

// Test if PROPFIND returns correct information for a directory inside $GLOBALS['ezc_webdav_testfolder']
// with UTF-8 in the directory name
// with HTTP_DEPTH = 0

// First upload the file in the folder $GLOBALS['ezc_webdav_testfolder']
$folder = new ezpObject( 'folder', (int)$GLOBALS['ezc_webdav_testfolderid'] );
$folder->name = "româneşte";
$folder->publish();

$_SERVER['REQUEST_URI'] = '/' . $GLOBALS['ezc_siteaccess'] . '/Content/' . $GLOBALS['ezc_webdav_testfolder'] . '/' . urlencode( "româneşte" );
$_SERVER['REQUEST_METHOD'] = 'PROPFIND';

$_SERVER['HTTP_DEPTH'] = '1';
$_SERVER['CONTENT_TYPE'] = 'application/xml';
$_SERVER['HTTP_CONTENT_LENGTH'] = '288';

// The PROPFIND request needs an XML body
$GLOBALS['ezc_post_body'] = <<<END
<?xml version="1.0" encoding="utf-8"?>
<propfind xmlns="DAV:"><prop>
<creationdate xmlns="DAV:"/>
<getcontentlength xmlns="DAV:"/>
<getcontenttype xmlns="DAV:"/>
<getlastmodified xmlns="DAV:"/>
<displayname xmlns="DAV:"/>
<executable xmlns="http://apache.org/dav/props/"/>
<resourcetype xmlns="DAV:"/>
<supportedlock xmlns="DAV:"/>
<checked-in xmlns="DAV:"/>
<checked-out xmlns="DAV:"/>
</prop></propfind>
END;

?>
