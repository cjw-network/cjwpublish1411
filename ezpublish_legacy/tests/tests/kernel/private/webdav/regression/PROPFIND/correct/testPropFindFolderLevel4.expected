Content-Type: text/xml; charset="utf-8"
Server: eZComponents/dev/ezcWebdavTransportWrapper
HTTP/1.1 207 Multi-Status

<?xml version="1.0" encoding="UTF-8"?>
<D:multistatus xmlns:D="DAV:">
  <D:response xmlns:default="http://apache.org/dav/props/">
    <D:href>http://@ezc_webdav_host@/@ezc_siteaccess@/Content/@ezc_webdav_testfolder@/rom%C3%A2ne%C5%9Fte</D:href>
    <D:propstat>
      <D:prop>
        <D:getcontentlength>4096</D:getcontentlength>
        <D:getlastmodified>Tue, 26 Aug 2008 11:45:49 +0000</D:getlastmodified>
        <D:creationdate>2008-08-26T11:37:50+0000</D:creationdate>
        <D:displayname>româneşte</D:displayname>
        <D:getcontenttype>httpd/unix-directory</D:getcontenttype>
        <D:resourcetype>
          <D:collection/>
        </D:resourcetype>
      </D:prop>
      <D:status>HTTP/1.1 200 OK</D:status>
    </D:propstat>
    <D:propstat xmlns:default="http://apache.org/dav/props/">
      <D:prop>
        <D:checked-in/>
        <D:checked-out/>
        <default:executable xmlns="http://apache.org/dav/props/"/>
      </D:prop>
      <D:status>HTTP/1.1 404 Not Found</D:status>
    </D:propstat>
  </D:response>
</D:multistatus>
